### PR TITLE
BACK-1652: adding initial prisma Label db model

### PR DESCRIPTION
**Pull request by katerinachinnappan**
## Goal

Created initial `Label` prisma db model:

```
id int(11)
externalId   varchar(255)
name           varchar(255)
createdt      datetime
createdBy   varchar(255)
updatedAt   datetime
updatedBy  varchar(255)
```

`externalId` and `name` are unique
index on `name` & `id`

Tickets:

[https://getpocket.atlassian.net/browse/BACK-1652](https://getpocket.atlassian.net/browse/BACK-1652
)

### CODE REVIEW

File: src/test/helpers.ts
Comment (Herraj) [inline src/test/helpers.ts:56]: Nice!

File: prisma/seed.ts
Comment (Herraj) [inline prisma/seed.ts:28]: nit: We've decided to keep the all lower case hyphen format for label names. Can we change `East Africa` to `region-east-africa` just to reflect how actual labels will be going forward?

**Review by Herraj — APPROVED** [review]

**Review by jpetto — APPROVED** [review]
this is looking really good!

i'm wondering if the deploy to dev is failing due to a migration issue. did you push to dev when the migration looked different? if so, that db may have gotten out of sync. happy to help rectify this if needed. just ping me in slack.

**Review by jpetto — APPROVED** [review]
re-approving after fixing the dev deployment

since merging this will run a db migration in prod, how about we wait until monday for a merge? just to be safe?

**Pull request merged**
